docs/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md

```diff
@@ -171,6 +220,150 @@ public class ExampleClass
+
+For more information, see [Quoted numbers](system-text-json-how-to.md#quoted-numbers).
+::: zone-end
+
+::: zone pivot="dotnet-core-3-1"
+`Newtonsoft.Json` can serialize or deserialize numbers represented by JSON strings (surrounded by quotes). For example, it can accept: `{"DegreesCelsius":"23"}` instead of `{"DegreesCelsius":23}`. To enable that behavior in <xref:System.Text.Json>, implement a custom converter like the following example. The converter handles properties defined as `long`:
+
+* It serializes them as JSON strings.
+* It accepts JSON numbers and numbers within quotes while deserializing.
+
+[!code-csharp[](snippets/system-text-json-how-to/csharp/LongToStringConverter.cs)]
+
+Register this custom converter by [using an attribute](system-text-json-converters-how-to.md#registration-sample---jsonconverter-on-a-property) on individual `long` properties or by [adding the converter](system-text-json-converters-how-to.md#registration-sample---converters-collection) to the <xref:System.Text.Json.JsonSerializerOptions.Converters> collection.
+::: zone-end
+
+### Specify constructor to use
+
+The `Newtonsoft.Json` `[JsonConstructor]` attribute lets you specify which constructor to call when deserializing to a POCO.
+
+::: zone pivot="dotnet-5-0"
+`System.Text.Json` also has a [[JsonConstructor]](xref:System.Text.Json.Serialization.JsonConstructorAttribute) attribute. For more information, see [Immutable types and Records](system-text-json-how-to.md#immutable-types-and-records).
+::: zone-end
+
+::: zone pivot="dotnet-core-3-1"
+<xref:System.Text.Json> supports only parameterless constructors. As a workaround, you can call whichever constructor you need in a custom converter. See the example for [Deserialize to immutable classes and structs](#deserialize-to-immutable-classes-and-structs).
+::: zone-end
+
+### Conditionally ignore a property
+
+`Newtonsoft.Json` has several ways to conditionally ignore a property on serialization or deserialization:
+
+* `DefaultContractResolver` lets you select properties to include or exclude, based on arbitrary criteria.
+* The `NullValueHandling` and `DefaultValueHandling` settings on `JsonSerializerSettings` let you specify that all null-value or default-value properties should be ignored.
+* The `NullValueHandling` and `DefaultValueHandling` settings on the `[JsonProperty]` attribute let you specify individual properties that should be ignored when set to null or the default value.
+
+<xref:System.Text.Json> provides the following ways to omit properties while serializing:
+
+::: zone pivot="dotnet-5-0"
+
+* The [[JsonIgnore]](system-text-json-how-to.md#exclude-individual-properties) attribute on a property causes the property to be omitted from the JSON during serialization.
+* The [IgnoreNullValues](system-text-json-how-to.md#exclude-all-null-value-properties) global option lets you exclude all null-value properties.
```

Comment:
```suggestion
* The [DefaultIgnoreCondition](system-text-json-how-to.md#exclude-all-null-value-properties) global option lets you exclude all null-value properties.
```

Maybe we should merge this with the info on L273.

FYI we are marking `IgnoreNullValues` as obsolete in .NET 6.0.